services/QuillLMS/app/lib/rule_feedback_history.rb

```diff
@@ -48,14 +48,18 @@ def self.feedback_history_to_json(f_h)
         entry: f_h.entry,
         highlight: f_h.metadata.instance_of?(Hash) ? f_h.metadata['highlight'] : '',
         session_uid: f_h.feedback_session_uid,
-        strength: f_h.feedback_history_ratings.order(updated_at: :desc).first&.rating
+        strength: f_h.feedback_history_ratings.max_by(&:updated_at)&.rating
```

Comment:
Using `order` on an AR relation triggers a separate DB call. Sorting in memory does not.